lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -1347,13 +1356,14 @@
                 var matcher = GetOption(options, "localeMatcher", "string", ["lookup", "best fit"], "best fit");
                 var timeZone = GetOption(options, "timeZone", "string", undefined, undefined);
```

Comment:
>var [](start = 16, length = 3)

nit: Nice opportunity to change all vars to let (and const where applicable) :-)

Reply:
Definitely planning on it, but that level of churn would delay productive work on Intl-ICU at the moment. Sometimes changing `var` to `let` causes hard-to-see issues (due to the permissiveness of `var`) and the Intl.js code is hard to test directly so it takes a long time to find out and that can be disruptive. I'll add a backlog task for this kind of refactoring.